css/site.styl

```diff
@@ -107,13 +114,29 @@ img
     opacity: 1
   100% 
     opacity: 1
-  
-input
-  height: 0
-  width: 0
-  opacity: 0
-  text-transform: uppercase
 
+// CONVERSATION FORM
+.conversation
+
+  #custom-form
```

Comment:
I see that this conversation class is a class for the main container. But what need is there to nest like this why not just use the `#custom-form` id?